src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -12,18 +11,69 @@ import {
   GEL_SPACING_DBL,
   GEL_SPACING_TRPL,
 } from '@bbc/gel-foundations/spacings';
-
+import Grid from '@bbc/psammead-grid';
 import { storyItem } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
-import { GridWrapper, GridItemConstrainedLarge } from '#lib/styledGrid';
+import {
+  GridWrapper as LegacyGridWrapper,
+  GridItemConstrainedLarge,
+} from '#lib/styledGrid';
+import { gelGridMargin } from '#app/lib/layoutGrid';
+
+const constrainedLargeGridColumns = {
+  group0: 1,
+  group1: 1,
+  group2: 1,
+  group3: 1,
+  group4: 6,
+  group5: 12,
+};
+
+const ConstrainedLargeGrid = ({ children, ...gridProps }) => (
+  <Grid
+    {...gridProps}
+    columns={constrainedLargeGridColumns}
+    margins={{
+      group0: true,
+      group1: true,
+      group2: true,
+      group3: true,
+      group4: false,
+      group5: false,
+    }}
+    startOffset={{
+      group0: 1,
+      group1: 1,
+      group2: 1,
+      group3: 1,
+      group4: 3,
+      group5: 6,
+    }}
+  >
+    {children}
+  </Grid>
+);
 
-const Wrapper = styled(GridItemConstrainedLarge)`
+ConstrainedLargeGrid.propTypes = {
+  children: node.isRequired,
+};
+
+const gridMarginSmall = css`
```

Comment:
It might be helpful to know that we don't need to use a `css` tagged template literal here, because these styles do not include functions.

https://styled-components.com/docs/api#css

This could be re-written as:

```js
const gridMarginSmall = `
  margin-bottom: ${GEL_SPACING_DBL};
  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
    margin-bottom: ${GEL_SPACING_TRPL};
  }
`;
```